### PR TITLE
React Popover kit!

### PR DESCRIPTION
**Pull request by thestephenmarshall**
#### Runway Ticket URL

Screen Shot 2020-02-13 at 8 22 56 AM

https://nitro.powerhrg.com/runway/backlog_items/NUX-600

#### Breaking Changes

- Renamed vanilla JS version of `Popover` to `PbPopover` to help prevent collision

#### How to Ninja test this (screenshots are really helpful)

1. Go to dev docs PB page and ensure that Popover is working
1. Check out the Corporate Directory page and ensure that the search filters popover is working

#### Checklist:

- ~~**SPECS** Please cover your changes with specs~~
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ def deployerVersion = "master-e4c01702624ac34f2663f894fbdb68a369b09629-588"
 def resources = [
   requestCpu: '1',
   limitCpu: '2',
-  requestMemory: '4Gi',
+  requestMemory: '6Gi',
```
Comment (thestephenmarshall) [inline Jenkinsfile]: Let's see how this works out for us while building. Given [@jasperfurniss updated the memory heap to 4GB](https://github.com/powerhome/playbook/commit/bf3ae54e0e0f4f914d7b2e360cc84b35f6a7aad4), we should need to increase this for headspace.

File: package.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
-    "@popperjs/core": "^2.0.0",
+    "popper.js": "^1.14.4",
```
Comment (thestephenmarshall) [inline package.json]: We should be using consistent versions for both this lib and `react-popper`

Comment (jasperfurniss) [reply package.json]: Hopefully someone is revamping that react-popper library for 2.0 popper support AND one day it will be super easy to use... 🤞 🤞

Comment (thestephenmarshall) [reply package.json]: @jasperfurniss https://github.com/popperjs/react-popper/issues/332

**Pull request merged**